### PR TITLE
[NUI][AT-SPI] Default accessibility support for Tizen.NUI.Components.Progress

### PR DESCRIPTION
**Pull request by m-bialota**
### Description of Change ###

[NUI][AT-SPI] Basic accessibility support for Tizen.NUI.Components.Progress

### API Changes ###

 - ACR: none

### CODE REVIEW

**Review by grzelewski — APPROVED** [review]

**Review by lol-github — CHANGES_REQUESTED** [review]
ATSPI notification on value change is missing

**Comment by dongsug-song** [comment]
@m-bialota 
could you check the @lol-github 's comment?
**"ATSPI notification on value change is missing"**
if this is fixed then please request a review again to @lol-github 
thank you.

**Comment by dongsug-song** [comment]
@m-bialota
could you check the @lol-github 's comment?
**"ATSPI notification on value change is missing"**
is this fixed?

**Comment by dongsug-song** [comment]
@m-bialota 
Hello, 
could you resolve conflicts or could you close this and upload new PR?

now I put the **NO MERGE** tag on this PR.
you can remove **NO MERGE** tag for yourself when this PR is reviewed and ready to be merged.
(**NO MERGE** tag has no restriction to submit PR or to edit this patch but just for notification and marking.
thank you.

**Comment by m-bialota** [comment]
> @m-bialota
> could you check the @lol-github 's comment?
> **"ATSPI notification on value change is missing"**
> is this fixed?

Yes, now PropertyChange AT-SPI event is emitted on dbus in case Progress object is highlighted and its value is changed.

**Review by Seoyeon2Kim — APPROVED** [review]
Looks good to me.

File: src/Tizen.NUI.Components/Controls/Progress.cs [hunk 1]
```diff
@@ -152,6 +152,7 @@ public class Progress : Control
         /// <since_tizen> 6 </since_tizen>
         public Progress() : base()
         {
+            Initialize();
```
Comment (a-bartak) [inline src/Tizen.NUI.Components/Controls/Progress.cs:155]: Don't we need this Initialize in two other ctors?

Comment (m-bialota) [reply src/Tizen.NUI.Components/Controls/Progress.cs:155]: You are right!

Comment (m-bialota) [reply src/Tizen.NUI.Components/Controls/Progress.cs:155]: done

**Review by a-bartak — APPROVED** [review]

**Pull request merged**